.github/workflows/nm-upload-assets-to-gcp.yml

```diff
@@ -68,6 +84,16 @@ jobs:
               with:
                   path: assets
 
+            - name: push wheel to pypi.org
+              # this workflow is only run if push-to-pypi is True, and we only
+              # want to push RELEASE wheels to the external pypi.org
+              if: ${{ inputs.wf_category }} == "RELEASE"
+              uses: neuralmagic/nm-actions/actions/publish_whl/action.yml@main
```

Comment:
Can we use nm-actions v1.0.0 instead of main here?

Reply:
good new capability since I submitted the PR.  thanks for suggesting it!